docker-compose-services/solr-5/README.md

```diff
@@ -45,4 +45,59 @@ DDEV has a note that it will clean up volumes that match the service name. Since
 
 You'll see two named something like `ddev-example_solr_opt` and `ddev-example_solr_var`.
 
+## Creating search indexes
+
+The `ddev start` process should create the 'dev' index the first time it the
+project is started. However, in certain circumstances this does not happen.
+The solution is to manually create the index from the server:
+
+    $ ddev ssh --service solr
+    $ bin/solr create_core -c dev -d /solr-conf
+
+This uses the Solr configuration files which ddev makes available from the path `/solr-conf` and creates an index named `dev`; it should result in the following output:
+
+    Copying configuration to new core instance directory:
+    /opt/solr/server/solr/dev
+    
+    Creating new core 'dev' using command:
+    http://localhost:8983/solr/admin/cores?action=CREATE&name=dev&instanceDir=dev
+    
+    {
+      "responseHeader":{
+        "status":0,
+        "QTime":1580},
+      "core":"dev"}
+
+To create a additional indexes just change the `dev` portion of the command
+above, e.g.
+
+    $ ddev ssh --service solr
+    $ bin/solr create_core -c myindex -d /solr-conf
+
+## Troubleshooting
+
+If the Solr service is not available to the client project (e.g. Drupal) using
+the expected URL format, the Solr index may not have been created correctly. It
+is possible to access Solr directly from the host OS by loading its full URL,
+e.g. `http://myproject.ddev.site:8983`.
+
+* Make note that the URL uses the HTTP protocol, not HTTPS.
+* Safari may insist in loading the URL as HTTPS, e.g,
```

Comment:
Just add HTTPS_EXPOSE=8984 like the [core example has](https://github.com/drud/ddev/blob/master/pkg/servicetest/testdata/TestServices/docker-compose.solr.yaml) and then use `https://myproject.ddev.site:8984` everywhere and this problem goes away.

Reply:
Good call, thank you. I've updated the README files to mention this.